backend/tracim_backend/lib/core/content.py

```diff
@@ -1907,17 +1908,33 @@ def update_file_data(
         item.revision_type = ActionDescription.REVISION
         return item
 
-    def check_upload_size(self, content_length: int):
+    def check_upload_size(self, content_length: int, workspace: Workspace) -> None:
+        self._check_size_length_limitation(content_length)
+        self._check_workspace_size_limitation(content_length, workspace)
+
+    def _check_size_length_limitation(self, content_length: int) -> None:
         # INFO - G.M - 2019-08-23 - 0 mean no size limit
         if self._config.LIMITATION__CONTENT_LENGTH_FILE_SIZE == 0:
-            return
+            pass
```

Comment:
`return`was a good choice. `self._config.LIMITATION__CONTENT_LENGTH_FILE_SIZE == 0` is a good reason to ensure no more code in this function must be executed.

Reply:
done.